src/flow.jl

```diff
@@ -218,3 +218,11 @@ function update(ssf::LateralSSF, network, frac_toriver)
         )
     end
 end
+
+@get_units @with_kw struct GroundwaterExchange{T}
+    Δt::T | "s"                         # model time step [s]
+    exfiltwater::Vector{T}              # Exfiltration [mm]  (groundwater above surface level, saturated excess conditions)
+    zi::Vector{T}                       # Pseudo-water table depth [mm] (top of the saturated zone)
+    to_river::Vector{T} | "m3 s-1"      # Part of subsurface flow [mm³ Δt⁻¹] that flows to the river
+    ssf::Vector{T} | "mm3 Δt-1"         # Subsurface flow [mm³ Δt⁻¹]
+end
```

Comment:
The units are the same in the comment but not the same in the annotation, m vs mm and s vs delta-t.

Reply:
Yes, the unit "m3 s-1" for  ``to_river`` was not correct, this should be the same as we use for the subsurface flow component (internal), so mm and delta-t.